gapis/gfxapi/gles/glsl/preprocessor/preprocessorImpl.go

```diff
@@ -314,7 +314,7 @@ func (p *preprocessorImpl) processDefine(args []TokenInfo) {
 	macro := args[0]
 
 	if _, ok := p.macros[macro.Token.String()]; ok {
-		p.err.Errorf("Macro '%s' already defined.", args[0].Token)
+		delete(p.macros, macro.Token.String())
```

Comment:
I think redefiniton is error only if the macro changes.  This code will die anyway.